GitCommands/Git/GitModule.cs

```diff
@@ -2774,13 +2776,18 @@ private static IReadOnlyList<GitItemStatus> GetSkipWorktreeFilesFromString(strin
             foreach (string line in lsString.LazySplit('\n', StringSplitOptions.RemoveEmptyEntries))
             {
                 char statusCharacter = line[0];
-
-                string fileName = line.SubstringAfter(' ');
-                GitItemStatus gitItemStatus = GitItemStatusConverter.FromStatusCharacter(StagedStatus.WorkTree, fileName, statusCharacter);
-                if (gitItemStatus.IsSkipWorktree)
+                const char SkippedStatus = 'S';
+                const char SkippedStatusAssumeUnchanged = 's';
+                if (statusCharacter is not SkippedStatus or SkippedStatusAssumeUnchanged)
```

Comment:
Hi, sorry for commenting on a closed PR but I just noticed that this condition does not look correct.

I assume the intention is to continue if statusCharacter is not one of 'S' or 's'? If so, then the condition should be:
`statusCharacter is not SkippedStatus and not SkippedStatusAssumeUnchanged`

As it is now, the loop will continue if statusCharacter is 's'.

Reply:
Parens missing?
```suggestion
                if (statusCharacter is not (SkippedStatus or SkippedStatusAssumeUnchanged))
```

Reply:
Actually, it is intentional. AssumeUnchanged are already added to the filelist.
There should be a comment though

Reply:
Interesting, I didn't know you could use _or_ in that way.